infra/aws/README.md

```
@@ -0,0 +1,250 @@
+   - [3-1. 사전 준비](#3-1-사전-준비-1회성)
+   - [3-2. 배포 명령](#3-2-배포-명령)
+   - [3-3. 배포 후 작업](#3-3-배포-후-작업-1회성)
+4. [동작 검증](#4-동작-검증)
+5. [모니터링](#5-모니터링)
+6. [롤백](#6-롤백)
+
+---
+
+## 1. 디렉토리 구조
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)